keyboards/hardlight/keymaps/default/config.h

```diff
@@ -0,0 +1,7 @@
+#undef  LEADER_TIMEOUT
+#define LEADER_TIMEOUT 250
+#define LEADER_PER_KEY_TIMING
+
+#pragma once
+
+// place overrides here
```

Comment:
```suggestion
#pragma once

#undef  LEADER_TIMEOUT
#define LEADER_TIMEOUT 250
#define LEADER_PER_KEY_TIMING

// place overrides here
```

`#pragma once` should begin the file.  Also please add a GPL2+ compatible license header.